.github/workflows/react-native-sdk.yml

```
@@ -0,0 +1,144 @@
+      with:
+        node-version: '18'
+        cache: 'yarn'
+        cache-dependency-path: sdk/runanywhere-react-native/yarn.lock
+
+    - name: Install dependencies
+      working-directory: ./sdk/runanywhere-react-native
+      run: yarn install --frozen-lockfile
+
+    - name: Detect available iOS simulator
+      id: detect-simulator
+      run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:10:32: Double quote to prevent globbing and word splitting

(shellcheck)